src/server/scripts/Northrend/zone_borean_tundra.cpp

```diff
@@ -539,8 +539,8 @@ enum Lurgglbr
     FACTION_ESCORTEE_A                  = 774,
     FACTION_ESCORTEE_H                  = 775,
 
-    SAY_START_1                         = 0,
-    SAY_START_2                         = 1,
+    SAY_START_1                         = 1,
+    SAY_START_2                         = 0,
```

Comment:
maybe add a comment here saying they are not in order on purpose, in case someone will wonder about it in the future